.github/workflows/labeler.yml

```
@@ -0,0 +1,12 @@
+name: Pull Request Labeler
+on:
+- pull_request_target
```

If I'm understanding what you want to do correctly, I think you either need:

```suggestion
  pull_request_target:
   branches:
     - main 
```

or perhaps a general `pull_request`?

[check the docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) as I don't think this is a valid workflow

---

This is a valid workflow, see [actions/labeler documentation](https://github.com/actions/labeler?tab=readme-ov-file#create-workflow). I have no further doubts about this comment if you mean to have the `main` branch be the only target branch of a PR that receives a `documentation` label.